direct/core/src/test/java/cz/o2/proxima/direct/storage/InMemStorage.java

```diff
@@ -737,44 +740,6 @@ private Executor executor() {
     }
   }
 
-  /** Estimator of watermark. */
-  public interface WatermarkEstimator extends Serializable, WatermarkSupplier {
```

Comment:
ALERT:
Not backward compatibile
Interface was modified little bit and moved to core module. 
@je-ik Can we live with it?

Reply:
Yes, we can do API breaking changes between versions, current release is `0.3.1`, this targets `0.4.x`.

Reply:
Easy to fix: https://gitlab.seznam.net/user-profile/platform/-/merge_requests/215/diffs